src/runtime/virtcontainers/network_darwin.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2023 Apple Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+
+	persistapi "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/persist/api"
+)
+
+var endpointNotSupported error = errors.New("Unsupported endpoint on Darwin")
+
+// DarwinNetwork represents a sandbox networking setup.
+type DarwinNetwork struct {
+	networkID         string
+	interworkingModel NetInterworkingModel
+	networkCreated    bool
+	eps               []Endpoint
+}
+
+func NewNetwork(configs ...*NetworkConfig) (Network, error) {
+	if len(configs) > 1 {
+		return nil, fmt.Errorf("Too many network configurations")
```

Comment:
Prefer `errors.New` if no formatting (also go idioms is to start errors lowercased)

Reply:
ack -- I'll keep non-idiomatic just based on it being the function name?

Reply:
Oh I mean the error string haha so "too many network configurations"*

Reply:
https://github.com/golang/go/wiki/Errors#errors